frontend/packages/dev-console/integration-tests/features/addFlow/upload-JAR-file.feature

```diff
@@ -0,0 +1,140 @@
+Feature: Upload JAR file
+              As a user, I want to upload a JAR file from using Upload JAR file card from Add page
+
+        Background:
+            Given user is at developer perspective
+              And user is at Add page
+              And user has created or selected namespace "aut-upload-jar-file"
+
+
+        @regression @manual
+        Scenario Outline: Upload JAR file from Add options
+            Given user is at the Add page
+             When user clicks on the Upload JAR file card
+              And user clicks on Browse in JAR file section
+              And user selects file to upload
+              And user selects appropriate Build image version
+              And user gives Application name as "sample-upload-app" and workload Name as "<name>"
+              And user selects resource type as "<resource_type>"
+              And user clicks create button
+             Then user will be redirected to Topology page
+              And user can see a toast notification of JAR file uploading with link to build logs
+              And user can see "<resource_type>" "<name>" in application "sample-upload-app" is created in topology
+
+        Examples:
+                  | name                 | resource_type     |
+                  | sample-yaml-upload-1 | Deployment        |
+                  | sample-yaml-upload-2 | Deployment Config |
+
+        @regression @manual
+        Scenario Outline: Upload JAR file through drag and drop from Add options
+            Given user is at the Add page
```

Comment:
Already it is covered as part of background, please remove it

Reply:
Change the Background part